codespell_lib/data/dictionary.txt

```diff
@@ -14815,8 +14877,14 @@ corresspondingly->correspondingly
 corressponds->corresponds
 corret->correct
 correted->corrected
+correting->correcting
```

Comment:
I guess occurrences of `correlating` would be rare at best, wouldn't they?

Reply:
This occurs a couple of times. I will add this, along with other suffixes.